controllers/storagecluster/cephcluster.go

```
@@ -116,9 +117,18 @@ func (obj *ocsCephCluster) ensureCreated(r *StorageClusterReconciler, sc *ocsv1.
 		return err
 	}
 
+	platform, err := r.platform.GetPlatform(r.Client)
+	if err != nil {
+		r.Log.Info("Failed to get platform")
```

should be logged as error.

---

shouldn't we return in case of error ?

---

We are changing the mon failover timeout for IBM platform only, but if it fails I don't think it should be a problem.
So maybe instead of just logging it, Throwing a warning would be better.

---

ok. so if we don't want to return error here, then warning should be fine and next if block should be skipped.

---

Logging the error